rebar.config

```diff
@@ -9,5 +9,5 @@
 {reset_after_eunit, true}.
 
 {deps, [
-        {lager, "2.0.3", {git, "git://github.com/basho/lager", {tag, "2.0.3"}}}
+        {lager, "2.*", {git, "git://github.com/basho/lager", {tag, "2.1.1"}}}
```

Comment:
It's too caremad, but this regexp allows any versions which have character `2` and 1+ subsequent characters, for example, `1.2.0`.